docs/resources/dhcp_fixed_address.md

```diff
@@ -55,6 +62,14 @@ resource "bloxone_dhcp_fixed_address" "example_fixed_address" {
   tags = {
     location = "site1"
   }
+  //dhcp options
+  dhcp_options = [
+    {
+      option_code  = data.bloxone_dhcp_option_codes.option_code.results.0.id
+      option_value = "1.1.1.1"
```

Comment:
can you change all `1.1.1.1` to `10.0.0.1` instead.

Reply:
done